### PR TITLE
Select ledger address

### PR DESCRIPTION
**Pull request by biafra23**
Handles #1647

**Missing**
- Tracking incomplete

Changes proposed in this pull request:
- Uses FakeDevice to get addresses
- Uses the following interface to obtain these addresses (subject to change):
```
interface LedgerAddressProvider {
    fun addressesForRange(derivationPath: String, range: LongRange): List<Solidity.Address>
    fun addressesForPage(derivationPath: String, start: Long, pageSize: Int): List<Solidity.Address>
}
```

Ledger Live | Ledger
------------ | -------
Screenshot_20210909-110143 | Screenshot_20210908-202610 

@gnosis/mobile-devs

### CODE REVIEW

**Comment by elgatovital** [comment]
named items don't look deactivated in the screenshot. It creates an impression they can be selected again for import

**Comment by biafra23** [comment]
@elgatovital The screenshot is outdated. I will update it.

File: app/src/main/java/io/gnosis/safe/ui/settings/owner/ledger/LedgerTabsFragment.kt [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+                        fromSeedPhrase = false,
+                        ownerKey = "",
+                        ownerSeedPhrase = "",
+                        ownerType = Owner.Type.LEDGER_NANO_X.value
+                    )
+                )
+            }
+            pager = LedgerPagerAdapter(this@LedgerTabsFragment)
+            ledgerContent.adapter = pager
+            TabLayoutMediator(ledgerTabBar, ledgerContent, true) { tab, position ->
```
Comment (elgatovital) [inline app/src/main/java/io/gnosis/safe/ui/settings/owner/ledger/LedgerTabsFragment.kt:82]: switching tabs should also change the owner that is selected

File: app/src/main/java/io/gnosis/safe/ui/settings/owner/ledger/LedgerTabsFragment.kt [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+            }
+        })
+
+        with(binding) {
+            backButton.setOnClickListener {
+                Navigation.findNavController(it).navigateUp()
+            }
+            nextButton.setOnClickListener {
+                findNavController().navigate(
+                    LedgerTabsFragmentDirections.actionOwnerSelectionFragmentToOwnerEnterNameFragment(
```
Comment (elgatovital) [inline app/src/main/java/io/gnosis/safe/ui/settings/owner/ledger/LedgerTabsFragment.kt:71]: derivationPath is missing

Comment (biafra23) [reply app/src/main/java/io/gnosis/safe/ui/settings/owner/ledger/LedgerTabsFragment.kt:71]: That is in the next PR: #1667

**Review by elgatovital — APPROVED** [review]

**Pull request merged**